oaknational/Oak-Web-Application
# fix: reset selection filters on page update

## Description
- Update unit visualiser to update correctly on data changes

## Issue(s)
- Changing phase in unit visualiser leads to 404

## How to test

1. Go to https://deploy-preview-2192--oak-web-application.netlify.thenational.academy/teachers/curriculum/maths-secondary/units
2. Click on `School Phase`
3. Click on `Secondary`
5. Click `View`
6. You should see the updated unit visualiser with the correct data

## Checklist
